Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -568,16 +534,23 @@ CGPathRef CGPathRetain(CGPathRef path) {
 }
 
 /**
-@Status Stub
-@Notes
+@Status Interoperable
 */
 void CGPathAddQuadCurveToPoint(CGMutablePathRef path, const CGAffineTransform* transform, CGFloat cpx, CGFloat cpy, CGFloat x, CGFloat y) {
-    UNIMPLEMENTED();
+    RETURN_IF(!path);
+
+    CGPoint endPoint = __CreateCGPointWithTransform(x, y, transform);
```

Comment:
The helper accounts for this.

---
In reply to: [88084937](https://github.com/Microsoft/WinObjC/pull/1384#discussion_r88084937) [](ancestors = 88084937)